src/arrayMethodSort.js

```diff
@@ -4,9 +4,41 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function (compareFunction = defoultSortFunction) {
```

Comment:
There is a typo in the parameter name 'defoultSortFunction'. It should be 'defaultSortFunction'.